api/v1beta1/foundationdbcluster_types_test.go

```diff
@@ -3584,4 +3597,48 @@ var _ = Describe("[api] FoundationDBCluster", func() {
 			})
 		})
 	})
+
+	Describe("routing configuration", func() {
+		var cluster *FoundationDBCluster
+		BeforeEach(func() {
+			cluster = &FoundationDBCluster{}
+		})
+
+		When("checking whether we need a headless service", func() {
+			It("respects the value in the flag", func() {
```

Comment:
```suggestion
			It("respects the headless service setting in the routing config", func() {
```

That is a little bit more verbose than before but also more precise instead of saying "flag".